requirements/prod.txt

```diff
@@ -29,42 +29,35 @@ cryptography==38.0.1
     #   -r requirements/prod.in
     #   authlib
     #   pyjwt
-    #   secretstorage
 docutils==0.16
     # via -r requirements/prod.in
-faker==12.3.3
+faker==15.0.0
     # via
     #   -r requirements/prod.in
     #   snowfakery
 fs==2.4.16
     # via -r requirements/prod.in
 github3-py==3.2.0
     # via -r requirements/prod.in
-greenlet==1.1.2
-    # via
-    #   snowfakery
-    #   sqlalchemy
+greenlet==1.1.3
+    # via snowfakery
 gvgen==1.0
     # via snowfakery
-idna==3.3
+idna==3.4
     # via
     #   requests
     #   snowfakery
 importlib-metadata==4.12.0
     # via keyring
-jeepney==0.8.0
-    # via
-    #   keyring
-    #   secretstorage
```

Comment:
Transitive dependency that gets pulled in depending on whether `pip-compile` is run on Linux.